script/race.yml

```diff
@@ -123,3 +123,19 @@
     - gypsy
     - Gipsy
     - gyp
+- type: simple
+  note: Replace racially-charged language with more accurate and inclusive words
+  considerate:
+    - blocklist
```

Comment:
Nice! It does seem that the first suggestions aren’t antonyms, whereas 2 and 3 are. Why not blacklist/passlist or wronglist/alrightlist?

Reply:
Good question. I could definitely make that a thing. My thinking was that blacklist->blocklist and whitelist->alrightlist are sort of homonyms. I think what you recommended make sense though!